src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -70,82 +70,82 @@ def test_backup_scenario(self, resource_group, vault_name, vm_name, storage_acco
+    #    self.cmd('backup vault create -n {vault3} -g {rg} -l {loc}', checks=[
+    #        self.check('name', '{vault3}'),
+    #        self.check('resourceGroup', '{rg}'),
+    #        self.check('location', '{loc}'),
+    #        self.check('properties.provisioningState', 'Succeeded')
+    #    ])
+
+    #    self.kwargs['vault4'] = self.create_random_name('clitest-vault', 50)
+    #    self.cmd('backup vault create -n {vault4} -g {rg} -l {loc}', checks=[
+    #        self.check('name', '{vault4}'),
+    #        self.check('resourceGroup', '{rg}'),
+    #        self.check('location', '{loc}'),
+    #        self.check('properties.provisioningState', 'Succeeded')
+    #    ])
+
+    #    number_of_test_vaults = 4
+
+    #    self.cmd('backup vault list', checks=[
+    #        self.check("length([?resourceGroup == '{rg}'])", number_of_test_vaults),
+    #        self.check("length([?name == '{vault1}'])", 1),
+    #        self.check("length([?name == '{vault2}'])", 1),
+    #        self.check("length([?name == '{vault3}'])", 1),
+    #        self.check("length([?name == '{vault4}'])", 1)
+    #    ])
+
+    #    self.cmd('backup vault list -g {rg}', checks=[
+    #        self.check("length(@)", number_of_test_vaults),
+    #        self.check("length([?name == '{vault1}'])", 1),
+    #        self.check("length([?name == '{vault2}'])", 1),
+    #        self.check("length([?name == '{vault3}'])", 1),
+    #        self.check("length([?name == '{vault4}'])", 1)
+    #    ])
+
+    #    storage_model_types = [e.value for e in StorageType]
+    #    vault_properties = self.cmd('backup vault backup-properties show -n {vault1} -g {rg} --query [0]', checks=[
+    #        JMESPathCheckExists("contains({}, properties.storageModelType)".format(storage_model_types)),
+    #        self.check('properties.storageTypeState', 'Unlocked'),
+    #        self.check('resourceGroup', '{rg}')
+    #    ]).get_output_in_json()
+
+    #    if vault_properties['properties']['storageModelType'] == StorageType.geo_redundant.value:
+    #        new_storage_model = StorageType.locally_redundant.value
+    #    else:
+    #        new_storage_model = StorageType.geo_redundant.value
+
+    #    self.kwargs['model'] = new_storage_model
+    #    self.cmd('backup vault backup-properties set -n {vault1} -g {rg} --backup-storage-redundancy {model}')
+    #    time.sleep(300)
+    #    self.cmd('backup vault backup-properties show -n {vault1} -g {rg} --query [0]', checks=[
+    #        self.check('properties.storageModelType', new_storage_model)
+    #    ])
+
+    #    self.cmd('backup vault delete -n {vault4} -g {rg} -y')
+
+    #    self.cmd('backup vault list', checks=[
+    #        self.check("length([?resourceGroup == '{rg}'])", number_of_test_vaults - 1),
+    #        self.check("length([?name == '{vault1}'])", 1),
+    #        self.check("length([?name == '{vault2}'])", 1),
+    #        self.check("length([?name == '{vault3}'])", 1)
+    #    ])
```

Comment:
Why do you block these tests?

Reply:
There's a known bug in SDK due to which GET/SET vault storage config is failing. The fix is supposed to be released around 5th July. Till then I've disabled test_backup_vaults and test_backup_crr as they were using GET/SET vault storage config calls. Tagging @chandrikagonuguntla for SDK-related details.

Reply:
So blocking these tests is to bypass the SDK issue, right? When the fix of SDK is released, will these tests be restored?

Reply:
Yes there was a regression in the SDK which was making a call to the wrong API version. This test can be disabled for now, it will be restored once the SDK fix is done

Reply:
OK